package.json

```diff
@@ -2,15 +2,28 @@
   "name": "@metamask/safe-event-emitter",
   "version": "3.0.0",
   "description": "An EventEmitter that isolates the emitter from errors in handlers",
-  "main": "index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
```

Comment:
Curious, is there a specific reason to prefer the ESM types?

Reply:
Better to use CJS actually. Build tools that support ESM will look at `exports` anyway to resolve the types.